explorer/e2e/testFunctions.ts

```diff
@@ -1,8 +1,22 @@
-import { expect, Page } from "@playwright/test";
+import { expect, Locator, Page, test } from "@playwright/test";
 import { TabDescription } from "./testInterfaces";
 
 /* eslint-disable sonarjs/no-duplicate-string  -- ignoring duplicate strings here */
 // Run the "Expect each tab to appear as selected when the corresponding url is accessed" test
+
+export const getFirstElementTextLocator = (
```

Comment:
It looks like this function is grabbing the first row in the table and then grabbing the text from the nth column; if so, can you rename this to `getFirstRowNthColumnCellTextLocator` or similar? It's a long name but I feel its gives a clearer idea of the function's intention.